application.py

```diff
@@ -7,33 +7,53 @@
 
 app = Flask(__name__)
 app.config['UPLOAD_FOLDER'] = 'uploads/'
+app.config['IMAGE_PROCESSOR'] = 'http://0.0.0.0:5000/api/upload_image'
+
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
     if request.method == 'POST':
+        create_directory()
         file = request.files['file']
         if 'file' not in request.files or file.filename == '':
             return render_template('base.html', template='index.html')
         elif file and allowed_file(file.filename):
             # Save file to file system
             filename = secure_filename(file.filename)
             file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
-            filesFound = {'file':open(app.config['UPLOAD_FOLDER'] + filename, 'rb')}
+            open_file = open(app.config['UPLOAD_FOLDER'] + filename, 'rb')
```

Comment:
Use the python `with` statement top make sure the file is closed. In this scenario, any crashes between now and the call to `close` will leave the file pointer open and cause issues.